subsystem/cluster_test.go

```diff
@@ -42,6 +42,29 @@ const (
 	minSuccessesInRow = 2
 )
 
+var clusterValidStatuses = [...]string{
```

Comment:
please add a comment that the order here is important

Reply:
I even added `Sorted` to their names.